### PR TITLE
[MODFISTO-243] - Define DTO schema for combination of fund code and e…

### PR DESCRIPTION
**Pull request by aleksandr2110**
## Purpose
https://issues.folio.org/browse/MODFISTO-243

## Approach
Define DTO schema for combination of fund code and expense class

## Pre-Merge Checklist
Before merging this PR, please go through the following list and take appropriate actions.

- Does this PR meet or exceed the expected quality standards?
  - [x] Examples exist for all schemas
  - [x] Descriptions exist for all schema properties
  - [x] All schemas pass raml-cop linting
- Does this introduce breaking changes?
  - [ ] Were there any schema changes?
  - [ ] There are no breaking changes in this PR.
  
If there are breaking changes, please **STOP** and consider the following:

- What other modules will these changes impact?
- Do JIRAs exist to update the impacted modules?
  - [ ] If not, please create them
  - [ ] Do they contain the appropriate level of detail?  Which endpoints/schemas changed, etc.
  - [ ] Do they have all they appropriate links to blocked/related issues?
- Are the JIRAs under active development?  
  - [ ] If not, contact the project's PO and make sure they're aware of the urgency.
- Do PRs exist for these changes?
  - [ ] If so, have they been approved?

Ideally all of the PRs involved in breaking changes would be merged in the same day to avoid breaking the folio-testing environment.  Communication is paramount if that is to be achieved, especially as the number of intermodule and inter-team dependencies increase.  

While it's helpful for reviewers to help identify potential problems, ensuring that it's safe to merge is ultimately the responsibility of the PR assignee.

### CODE REVIEW

**Comment by grabsefx** [comment]
Please sign Contributor License Agreement https://cla-assistant.io/folio-org/acq-models?pullRequest=322

File: mod-finance/examples/fund_code_vs_expense_classes_type.sample [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+  fundCode : "Fund1",
```
Comment (grabsefx) [inline mod-finance/examples/fund_code_vs_expense_classes_type.sample:2]: Invalid declaration of json field. please change it to "fundCode"

**Review by AndreiMPV — APPROVED** [review]

File: mod-finance/examples/fund_code_vs_expense_classes_type_collection.sample [hunk 2]
```diff
@@ -0,0 +1,15 @@
+{
+    delimiter : ":",
+    [
+        {
+            fundCode : "Fund1",
```
Comment (grabsefx) [inline mod-finance/examples/fund_code_vs_expense_classes_type_collection.sample:5]: Please specifiy sample values from existing sample data to be more clear

File: mod-finance/examples/fund_code_vs_expense_classes_type_collection.sample [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+            ledgerCode : "Ledger1",
+            [Fund1, Fund1:ExpClassPrn, Fund1:ExpClassElec]
+        },
+        {
+            fundCode : "Fund2",
+            ledgerCode : "Ledger2",
+            [Fund2, Fund2:ExpClassPrn, Fund2:ExpClassElec]
```
Comment (grabsefx) [inline mod-finance/examples/fund_code_vs_expense_classes_type_collection.sample:12]: Invalid declaration of json fields and values

**Review by grabsefx — CHANGES_REQUESTED** [review]
Please fix the schema according to comments above

**Review by grabsefx — APPROVED** [review]

**Pull request merged**